filebeat/docs/load-balancing.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[load-balancing]]
+== Configuring Load Balancing
+
+//REVIEWERS: Which outputs support load balancing? I only see the loadbalance option described under Logstash and Redis. Are any other outputs supported?
+
+Filebeat provides configuration options that you can use to fine
+tune load balancing when sending events to multiple hosts. 
+
+To enable load balancing, you specify `load_balance: true` when you configure
+the output. For example:
+
+[source,yaml]
+-------------------------------------------------------------------------------
+output.logstash:
+  hosts: ["localhost:5044", "localhost:5045"]
+  loadbalance: true
+-------------------------------------------------------------------------------
+
+Filebeat can perform load balancing in a few different modes:
+
+* **Send events to one host after another:**
++
+By default, when you configure Filebeat to send events to multiple hosts
+(`load_balance: true`), Filebeat will send the events to one host after
```

Comment:
@urso I'm not sure what you want me to change or add here. I don't think I need to mention `publish_async` until later because the default is `false`. Also, I'm not sure if it makes sense to recommend changing `spool_size` until later in this topic.  I've changed the intro to say, "Filebeat can send events in a few different modes" since it sounds like the first approach is not truly load balancing (even though the `loadbalance` option is enabled). I'll push my changes in a bit, and maybe you can tell me specifically what I need to add or change to make the section right.